<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fixupgames</class>
 <widget class="QDialog" name="fixupgames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fixup Games editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Images Not in Database</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="notDatabaseTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="scanMissingImagesButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>383</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="fixImagesButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Try Fixup Images</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Missing Status</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="statusTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="scanMissingStatusButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>383</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="fixStatusButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Try Fixup Status</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Fixing Firmware</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeView" name="fwTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="scanMissingFWButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>256</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="fixFirmwareButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Try Fixup Status</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>fixStatusButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>fixupstatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanMissingStatusButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>scanstatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanMissingImagesButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>scanImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fixImagesButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>fixupImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scanMissingFWButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>scanFW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fixFirmwareButton</sender>
   <signal>pressed()</signal>
   <receiver>fixupgames</receiver>
   <slot>fixupFW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fixupstatus()</slot>
  <slot>scanstatus()</slot>
  <slot>scanImages()</slot>
  <slot>fixupImages()</slot>
  <slot>scanFW()</slot>
  <slot>fixupFW()</slot>
 </slots>
</ui>
